includes/wikia/nirvana/WikiaResponse.class.php

```diff
@@ -28,7 +28,7 @@ class WikiaResponse {
 	/**
 	 * Output formats
 	 */
-	const FORMAT_RAW = 'raw';
+	const FORMAT_RAW = 'raw';	// deprecated (PLATFORM-2770)
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Move this trailing comment on the previous empty line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS139)